eZ/Publish/API/Repository/Tests/LocationServiceTest.php

```diff
@@ -2202,7 +2202,7 @@ public function testMoveSubtreeUpdatesSubtreePropertiesHidden()
             $expected[$id]['depth'] = $properties['depth'] + 2;
             $expected[$id]['pathString'] = str_replace(
                 $locationToMove->pathString,
-                "{$newParentLocation->pathString}" . $this->parseId('location', $locationToMove->id) . '/',
+                (string)$newParentLocation->pathString . $this->parseId('location', $locationToMove->id) . '/',
```

Comment:
@kalessil These turn out a bit strange when there is concatenation going on, or?

Reply:
In this specific file, both variants are quite questionable ;) Concatenations should cover to string conversion.
So if maintainers (having better code insight) proposing to get rid of casting completely, I'll gladly do so.

Reply:
@kalessil cast here is not necessary as it's always gonna be a string, let's just remove surrounding `"{ }"` in all relevant places, because indeed that's an overkill.

Reply:
Ok, will do.